views/keyword/show.html

```diff
@@ -2,7 +2,12 @@
   <h2 class="keyword-detail__title">
     {{ .KeywordPresenter.Keyword }} keyword search result report
   </h2>
+
   <div class="keyword-detail__body">
     {{ template "keyword/_counting_result_report.html" . }} {{ template "keyword/_url_result_report.html" . }}
   </div>
+
+  <div class="keyword-detail__footer text-center">
+    <a href="/keyword/{{ .KeywordPresenter.KeywordId }}/render-html" class="btn btn-primary">Render Search Result HTML</a>
```

Comment:
@junan You can use `uerfor` here 🤟

Reply:
As I said before in other PR, I current one as it is simple and redable